buildSrc/src/main/kotlin/dependencies/Libraries.kt

```diff
@@ -24,6 +23,7 @@ object Libraries {
     val WCA_I18N = "com.github.thewca:wca_i18n:${Versions.WCA_I18N}"
     val GOOGLE_APPENGINE_GRADLE = "com.google.cloud.tools:appengine-gradle-plugin:${Versions.GOOGLE_APPENGINE_GRADLE}"
     val GOOGLE_CLOUD_STORAGE = "com.google.cloud:google-cloud-storage:${Versions.GOOGLE_CLOUD_STORAGE}"
+    val TNOODLE_SCRAMBLES = "org.worldcubeassociation.tnoodle:lib-scrambles:${Versions.TNOODLE_SCRAMBLES}"
```

Comment:
Another very naive question: is there a way to analyse our dependencies and see which one are used/unused?
(maybe you did run something like that to remove gwtexporter :))

Reply:
Well, since all dependencies are refactored into variables here, you can just use your favorite IDE's "show unused variables" inspection.
Please note, however, that with the way Gradle works these constants are just *links to the dependency*. Having an "unused" dependency in Gradle is as unharmful as pasting the URL `https://worldcubeassociation.org` in some arbitrary document.